static/js/components/widgets/ResourcePickerDialog.tsx

```diff
@@ -29,59 +41,99 @@ interface Props {
   title: string
   id: string
-  contentType: "resource" | "page"
+  contentType:
+    | typeof CONTENT_TYPE_RESOURCE
+    | typeof CONTENT_TYPE_PAGE
+    | typeof CONTENT_TYPE_COURSE_COLLECTION
+    | typeof CONTENT_TYPE_RESOURCE_COLLECTION
   resourcetype: string | null
   embeddable: boolean
+  singleColumn: boolean
+  sourceWebsiteName?: string
 }
-const RESOURCE_PICKER_TABS: ResourceTabSettings[] = [
+
+const RESOURCE_PICKER_TABS_MAIN: ResourceTabSettings[] = [
   {
     title:        "Documents",
     id:           RESOURCE_TYPE_DOCUMENT,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_DOCUMENT,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: true
   },
   {
     title:        "Videos",
     id:           RESOURCE_TYPE_VIDEO,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_VIDEO,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: false
   },
   {
     title:        "Images",
     id:           RESOURCE_TYPE_IMAGE,
     contentType:  CONTENT_TYPE_RESOURCE,
     resourcetype: RESOURCE_TYPE_IMAGE,
-    embeddable:   true
+    embeddable:   true,
+    singleColumn: false
   },
   {
     title:        "Pages",
     id:           CONTENT_TYPE_PAGE,
     contentType:  CONTENT_TYPE_PAGE,
     resourcetype: null,
-    embeddable:   false
+    embeddable:   false,
+    singleColumn: true
   }
 ]
+const RESOURCE_PICKER_TABS_MORE: ResourceTabSettings[] = [
+  {
+    title:             "Course Collections",
+    id:                CONTENT_TYPE_COURSE_COLLECTION,
+    contentType:       CONTENT_TYPE_COURSE_COLLECTION,
+    resourcetype:      null,
+    embeddable:        false,
+    singleColumn:      true,
+    sourceWebsiteName: "ocw-www"
+  },
+  {
+    title:             "Resource Collections",
+    id:                CONTENT_TYPE_RESOURCE_COLLECTION,
+    contentType:       CONTENT_TYPE_RESOURCE_COLLECTION,
+    resourcetype:      null,
+    embeddable:        false,
+    singleColumn:      true,
+    sourceWebsiteName: "ocw-www"
+  }
+]
+const RESOURCE_PICKER_TABS = [
+  ...RESOURCE_PICKER_TABS_MAIN,
+  ...RESOURCE_PICKER_TABS_MORE
+]
 
 const modeText = {
   [RESOURCE_EMBED]: {
-    headerText: "Resources",
+    title:      "Embed Resource",
     acceptText: "Embed resource"
   },
   [RESOURCE_LINK]: {
-    headerText: "Resources & Pages",
+    title:      "Link to Content",
     acceptText: "Add link"
   }
 }
 
 export default function ResourcePickerDialog(props: Props): JSX.Element {
   const { mode, isOpen, closeDialog, insertEmbed } = props
 
-  const currentTabs = RESOURCE_PICKER_TABS.filter(
+  const tabsMain = RESOURCE_PICKER_TABS_MAIN.filter(
     tab => mode !== RESOURCE_EMBED || tab.embeddable
   )
-  const [activeTab, setActiveTab] = useState(RESOURCE_TYPE_IMAGE)
+  const tabsMore = RESOURCE_PICKER_TABS_MORE.filter(
+    tab => mode !== RESOURCE_EMBED || tab.embeddable
+  )
+  const tabs = [...tabsMain, ...tabsMore]
```

Comment:
is it possible that this would be a little clearer?

```suggestion
  const tabs = mode === RESOURCE_EMBED
    ? [...tabsMain, ...tabsMore].filter(tab => tab.embeddable)
    : [...tabsMain, ...tabsMore]
```

idk, just took me a minute to follow what's going on so thought maybe we could simplify